gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ArtifactoryController.java

```diff
@@ -28,11 +29,9 @@
 @RequestMapping("/artifactory")
 public class ArtifactoryController {
 
+  @Autowired(required = false)
   private IgorService igorService;
 
-  public ArtifactoryController(IgorService igorService) {
```

Comment:
We prefer to use autowired constructors over properties. You can mark a bean as optional by typing it as such:

```
@Autowired
public ArtifactoryController(Optional<IgorService> igorService) {
```